### ISSUE TITLE
How to change the week header into days in the chart

### ISSUE THREAD

**Issue by vijaykumawat0578** [JayeshB92/Gantt_Chart_Library#1]
I implemented your code into my project but i want to change the view acording to days instead of week.
can you help me doing that?

**Comment by JayeshB92** [comment]
Yeah Sure, Can you provide me with your expected UI, So that I can provide you exactly what you are in need of..

**Comment by vijaykumawat0578** [comment]
Thanks for your concern, I want to achieve this kind of header [http://prntscr.com/l3evbc](url).

can you help me in this?

**Comment by JayeshB92** [comment]
Thanks for details, Definitely we can have such dates available with headers will update you soon.

Generally, With most of  the scenarios people have planning in weeks or months, based on that Library was designed.

Then too will look after the same and update you soon.

**Comment by xvbo** [comment]
I have the same requirement. If there is update, please contact me. Thank you.

**Comment by vijaykumawat0578** [comment]
> Thanks for details, Definitely we can have such dates available with headers will update you soon.
> 
> Generally, With most of the scenarios people have planning in weeks or months, based on that Library was designed.
> 
> Then too will look after the same and update you soon.

Have you worked on this?
As I'm failed to achive this yet after spending too much time on it.

**Comment by SachinTanpure** [comment]
I have the same requirement. i need to show day instead of week . If there is any update, please contact me. Thank you.
